packages/ui/src/components/layout/Layout.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { twOverrides } from '@/overrides';
+
+export const layoutDef = twOverrides(
+  {
+    header: 'h-16',
+    root: 'flex flex-col h-screen',
+    wrapper: 'flex flex-1 overflow-hidden',
+    sideBar: 'h-full w-64 overflow-y-auto',
+    content: 'flex-1 overflow-y-auto',
+  },
```

Comment:
je potřeba to připravit bez overflow... připravím PR ale ten zbytek prosím oprav.